app/models/eight_key.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class EightKey < ActiveRecord::Base
+  include Loadable, Exportable
+
+  validate :validate_ope_and_cross
+
+  MAP = {
+    'institution of higher education' => { institution: InstitutionConverter },
+    'city' => { city: BaseConverter },
+    'state' => { state: StateConverter },
+    'opeid' => { ope: OpeConverter },
+    'ipeds_id' => { cross: CrossConverter },
+    'notes' => { notes: BaseConverter }
+  }.freeze
+
+  def validate_ope_and_cross
+    return if ope.present? || cross.present?
```

Comment:
Should this method have a different name? ope_and_cross vs ope || cross logic. Perhaps `validate_either_ope_or_cross` or similar would be more descriptive of what's actually happening.